media-gfx/alembic/alembic-1.7.9.ebuild

```diff
@@ -14,10 +14,7 @@ SRC_URI="https://github.com/${PN}/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
 LICENSE="BSD"
 
 SLOT="0"
-# TODO: ~x86 currently depends on new =dev-python/pyilmbase-2.3.0 which has
-# ~x86 keyword. As soon as it's updated in the tree, the keyword can be
-# added here.
-KEYWORDS="~amd64"
+KEYWORDS="~amd64 ~x86"
```

Comment:
keywording should be left to x86 arch team, which at least is going to make an appropriate separate commit.

Reply:
I tested it in a 32-bit chroot, because I don't have a real ~x86 machine available.